### DISCUSSION TITLE
How do you escape a dollar sign?

### DISCUSSION THREAD

**Discussion by eggbean** [starship/starship#3182] [Q&A] [answered]
```
success_symbol = "[$$](bold green)"

[WARN] - (starship::modules::character): Error in module `character`:
 --> 1:3
  |
1 | [$$](bold green)
  |   ^---
  |
  = expected variable_name
```

I have tried escaping the dollar sign with a backslash and a second dollar sign, but neither work. I want this to remind me that I am in bash. How do I escape $?

**Comment by andytom** [comment, marked as answer]
Hi, I think you need to either need to use a double `\` or switch to use raw strings `'` and a single `\` e.g

```toml
success_symbol = "[\\$](bold green)" 
# or 
success_symbol = '[\$](bold green)'
```
